### PR TITLE
Install kwin when installing kde

### LINKED ISSUE #3651: kwin window manager missing from KDE target
#### Result of `sudo edit-chroot -all`:
    
    name: artful
    encrypted: no
    crouton: version 1-20171230152552~master:de1361af
    release: artful
    architecture: amd64
    xmethod: xorg
    targets: lxde,kde,e17,xorg,extension,gnome,xfce,gnome-desktop,unity,xiwi
    host: version 9901.74.2017_11_11_1607 (Developer Build - vanilla) developer-build amd64-vanilla
    kernel: Linux localhost 4.4.79 #3 SMP PREEMPT Mon Nov 6 19:42:08 GMT 2017 x86_64 x86_64 x86_64 GNU/Linux
    freon: yes

#### Please describe your issue:
When installing KDE into a chroot, the kwin package is not installed, so there is no window manager installed automatically. This can be fixed by adding `apt-get install kwin` to the kde target. I would have submitted a pull request for this if I knew where to put the fix.

#### If known, describe the steps to reproduce the issue:
1. Install KDE with `sudo crouton -u -n <chroot> -t kde`
2. Open KDE with `sudo startkde`
3. Notice there are no window titles

### PR DESCRIPTION
**Pull request by MCJack123**
Fixes #3651

### CODE REVIEW

**Comment by MCJack123** [comment]
I had signed it before, but I forgot to put in my GitHub name.

File: targets/kde [hunk 1]
```diff
@@ -9,7 +9,7 @@ CHROOTBIN='crouton-noroot startkde'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-install --minimal kde-baseapps kde-runtime plasma-desktop pulseaudio \
+install --minimal kde-baseapps kde-runtime plasma-desktop pulseaudio kwin \
```
Comment (dnschneid) [inline targets/kde]: kwin package doesn't exist on trusty (which isn't EOL'd quite yet), so please go through the various releases and make sure kwin exists on them.

**Comment by MCJack123** [comment]
I moved the kwin install to run only if it is xenial/stretch or newer.

**Comment by dnschneid** [comment]
Dunno why I took so long on this one.  Thank you!

**Pull request merged**